### PR TITLE
Sql hybrid cloud hol

### PR DESCRIPTION
**Pull request by paulopsgility**
I need to add alt text to some images but functionally everything is ready for testing

### CODE REVIEW

**Comment by DawnmarieDesJardins** [comment]
@paulopsgility 
Hi Paul, 
SME review for this workshop is complete.  Please finalize the update and assign it to me when it's ready for QC/merge.

File: Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md [hunk 1]
```diff
@@ -0,0 +1,976 @@
...
+<!-- TOC -->
+
+- [SQL Server Hybrid Cloud hands-on lab step-by-step](#sql-server-hybrid-cloud-hands-on-lab-step-by-step)
+  - [Abstract and learning objectives](#abstract-and-learning-objectives)
+  - [Overview](#overview)
+  - [Solution architecture](#solution-architecture)
+  - [Requirements](#requirements)
+    - [Help references](#help-references)
+  - [Exercise 1: SQL Backup solution](#exercise-1-sql-backup-solution)
+    - [Task 1: Create an Azure Storage Account](#task-1-create-an-azure-storage-account)
```
[the remark this answers is not in the document]
Comment (paulopsgility) [reply Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md]: Yes, this is part of the "Before the HOL" part of the lab.

File: Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md [hunk 2]
```diff
@@ -0,0 +1,976 @@
...
+    GO
+    RECONFIGURE
+    GO
+    EXEC sp_configure 'Agent XPs', 1
+    GO
+    RECONFIGURE
+    GO
+    ```
+
+5.  Paste the T-SQL code you copied at the end of the previous task into the query window replacing the existing code and click **Execute**. This code creates the new SQL identity with a Shared Access Signature for your storage account.
```
[the remark this answers is not in the document]
Comment (paulopsgility) [reply Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md]: Added the example.

File: Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md [hunk 3]
```diff
@@ -0,0 +1,976 @@
...
+
+7.  Execute the code using PowerShell ISE. Make sure you change the **\$storageAcctName = \"\[unique storage account name\]\"** field to a unique storage account name across Azure prior to execution. 
+
+    >**Note**: Due to differences between markdown and PowerShell formatting requirements, you may need to remove the white space prior to the **\"\@** near the end of the file.
+
+8.  Save the T-SQL code generated between the **Begin TSQL Script** and **End TSQL Script** in your PowerShell ISE output after execution into a notepad file. This code creates an identity using a Shared Access Signature (SAS) to a container in the storage account and configures managed backup when executed.
+
+### Task 2: Configure managed backup in SQL Server
+
+1.  From within your SQL Server virtual machine, launch SQL Server Management Studio and connect to the local database instance.
```
[the remark this answers is not in the document]
Comment (paulopsgility) [reply Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md]: Added additional clarity for this instruction.

**Comment by paulopsgility** [comment]
@DawnmarieDesJardins 
All feedback incorporated.

File: Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md [hunk 4]
```diff
@@ -0,0 +1,952 @@
...
+
+6. Make note of the server name you chose above. It will be used in a later task.
+
+7. After the SQL Server logical server has deployed, navigate to it and select **Firewalls and virtual networks** from the security menu on the left.
+
+    ![Security menu with Firewalls and virtual networks highlighted.](images/hands-on-lab/2019-03-20-11-56-21.png "Security menu")
+
+8. Add a new firewall rule by clicking **+Add client IP** and then clicking **Save**.
+
+    ![](images/hands-on-lab/2019-03-20-12-01-55.png "Firewall configuration")
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md]: Missing alt-text

File: Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md [hunk 5]
```diff
@@ -0,0 +1,952 @@
...
+
+7. On the select virtual machines blade, select the **CloudShopWeb** virtual machine for replication. We will only use Azure Site Recovery to protect the web servers. Click the **OK** button.
+
+    ![CloudShopWeb is highlighted in the select virtual machines window.](images/hands-on-lab/2019-03-26-00-02-35.png "Select virtual machines")
+
+8. On the Configure settings blade, set the Target location to the location of your CloudShop2 resource group. 
+
+9. Click on the **Customize** link.
+
+    ![](images/hands-on-lab/2019-03-26-03-26-07.png)
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md]: missing alt-text

File: Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md [hunk 6]
```diff
@@ -0,0 +1,952 @@
...
+
+10. Change the target resource group to **CloudShop2**.
+
+    ![](images/hands-on-lab/2019-03-26-03-27-29.png)
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md]: missing alt-text

File: Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md [hunk 7]
```diff
@@ -0,0 +1,952 @@
...
+
+11. Click **Create target resources**. 
+
+    **Note**: Do not close the blade. It will close by itself after the target resources are created (2-3 minutes).
+
+    ![](images/hands-on-lab/2019-03-26-00-06-44.png)
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md]: missing alt-text

File: Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md [hunk 8]
```diff
@@ -0,0 +1,952 @@
...
+
+12. On the Enable Replication blade, click the **enable replication** button. 
+
+13. Several Site Recovery jobs will be initiated which are creating the replication policy as well as the target resources to be used during a failover. Click on **Site recovery jobs** to view the job progress.
+
+    ![](images/hands-on-lab/2019-03-26-00-52-02.png)
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md]: missing alt-text

File: Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md [hunk 9]
```diff
@@ -0,0 +1,952 @@
...
+
+14. Once all the jobs are successful, click on **Replicated items** under **Protected Items** to view the status of the initial replication.
+    
+    ![](images/hands-on-lab/2019-03-26-00-53-16.png)
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md]: missing alt-text

File: Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md (hunk 6 above)
Comment (paulopsgility) [reply Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md]: Updated

File: Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md (hunk 7 above)
Comment (paulopsgility) [reply Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md]: Updated

File: Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md (hunk 8 above)
Comment (paulopsgility) [reply Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md]: Updated

File: Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md (hunk 4 above)
Comment (paulopsgility) [reply Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md]: Updated

File: Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md (hunk 5 above)
Comment (paulopsgility) [reply Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md]: Updated

File: Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md (hunk 9 above)
Comment (paulopsgility) [reply Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md]: Updated

**Pull request merged**